README.md

```diff
@@ -1,3 +1,86 @@
-# Developer Rubric Discord Bot
+<div id="top"></div>
+
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  <a href="https://www.semasoftware.com/">
+    <img src="https://cdn.discordapp.com/icons/972616580078907442/3660c78f24a43ee25aa2e1a313d18424.webp?size=160" alt="Logo" width="80" height="80" >
+  </a>
+
+  <h3 align="center">Developer Rubric Discord Bot</h3>
+
+  <p align="center">
+    <!-- <a href="">View Demo</a>
+    . -->
+    <a href="https://github.com/Semalab/developer-skills-matrix-discord-bot/issues/new?assignees=nhcarrigan&labels=%F0%9F%9A%A6+status%3A+awaiting+triage&template=bug_report.yml&title=%5BBUG%5D+-+">Report Bug</a>
+    ·
+    <a href="https://github.com/Semalab/developer-skills-matrix-discord-bot/issues/new?assignees=nhcarrigan&labels=%F0%9F%9A%A6+status%3A+awaiting+triage&template=feature_request.yml&title=%5BFEAT%5D+-+">Request Feature</a>
+  </p>
+</div>
+
+## About The Project
+
+<!-- [![Developer Rubric Discord Bot][product-screenshot]](https://example.com) -->
 
 This is a Discord bot which allows you to assess your skills with our [developer rubric](https://github.com/Semalab/developer-rubric) directly on Discord.
```

Comment:
The link is outdated. We should keep it up-to-date.
```suggestion
This is a Discord bot that allows you to assess your skills with our [developer rubric](https://github.com/Semalab/developer-skills-matrix) directly on Discord.
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient